ts/api/api.ts

```diff
@@ -362,3 +362,21 @@ class InitializeOnDemandApi implements OnDemandOpenYoloApi {
 }
 
 export const openyolo: OnDemandOpenYoloApi = new InitializeOnDemandApi();
+
+// Export the public methods.
+const windowAsAny = window as any;
+windowAsAny['openyolo'] = openyolo;
+InitializeOnDemandApi.prototype['setProviderUrlBase'] =
```

Comment:
tsc does preserve comments, but the rollup phase deletes them... So it seems to be the only solution for now. And it's actually the method recommended by the Closure doc: https://developers.google.com/closure/compiler/docs/api-tutorial3#export.